package.json

```diff
@@ -25,4 +30,4 @@
     "eslint-plugin-jsx-a11y": "^6.2.1",
     "eslint-plugin-react": "^7.12.4"
   }
-}
+}
```

Comment:
Good job on these @pbkabali , could you take care of the EOLs

Reply:
Thanks very much @codjoero. Somehow, format on-save is removing this line. I should just disable it for now and investigate. I already have prettier disabled too.